src/Aspire.ProjectTemplates.net8/templates/aspire-starter/Aspire-StarterApplication.1.Web/wwwroot/lib/bootstrap/dist/css/bootstrap-grid.css

```diff
@@ -0,0 +1,4085 @@
+/*!
```

Comment:
I think all of the files in this directory are not based on main yet and should all be removed. We no longer have a Aspire.ProjectTemplates.net8 project :smiley:

Reply:
Ah they all moved under the one templates project right? Let me move these files.